Rakefile

```diff
@@ -174,12 +189,102 @@ desc 'update tmate'
 file 'public/files/tmate-static-linux-amd64.tar.gz' => 'public/files' do
   latest_release = latest_release_for 'tmate-io/tmate'
   logger.info "Latest tmate release is #{latest_release}"
-  fetch_githubusercontent_file "tmate-io/tmate/releases/download/#{latest_release}/tmate-#{latest_release}-static-linux-amd64.tar.gz", "github.com", 'tmate-static-linux-amd64.tar.gz'
+  fetch_githubusercontent_file(
+    File.join(
+      'tmate-io/tmate/releases/download',
+      latest_release,
+      "tmate-#{latest_release}-static-linux-amd64.tar.gz",
+    ),
+    host: "github.com", to: 'tmate-static-linux-amd64.tar.gz'
+  )
 end
 
 desc 'update rustup'
 file 'public/files/rustup-init.sh' => 'public/files' do
-  fetch_githubusercontent_file "", "sh.rustup.rs", "rustup-init.sh"
+  fetch_githubusercontent_file '', host: 'sh.rustup.rs', to: 'rustup-init.sh'
+end
+
+desc 'update raw gimme versions'
+file 'public/files/gimme-versions-binary-linux' => 'tmp' do
+  fetch_githubusercontent_file(
+    'travis-ci/gimme/master/.testdata/sample-binary-linux',
```

Comment:
Ahhh, this is the line I couldn't find on my previous pass through this file -- just to confirm, this essentially makes https://github.com/travis-ci/gimme/blob/31c7d80fddc8058c490d36ff4da1d70dd5a93aa8/.testdata/sample-binary-linux the source-of-truth for which version of Go is the "latest" of each release track?